### DISCUSSION TITLE
How do I make Watchtowers slack notifications less verbose?

### DISCUSSION THREAD

**Discussion by marefati110** [containrrr/watchtower#630] [Q&A] [answered]
i use watchtower for update my containers and work and sedn notification by slack
but when there is no update send Unable to _update container /X Proceeding to next._ every 10 secound
how i can fix that
image are in a private docker registery 

i use this docker-compose

```docker-compose
  watchtower:
    image: containrrr/watchtower
    restart: always
    container_name: watchtower
    environment:
      WATCHTOWER_NOTIFICATIONS: slack
      WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL: xxx
      WATCHTOWER_NOTIFICATION_SLACK_CHANNEL: xxx
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /root/.docker/config.json:/config.json
    command: back-master front-master --interval 10 --debug

```

thanks :D

**Comment by zoispag** [comment]
Hi there,

You have an interval `--interval 10` which explains the message every 10 seconds and you have `--debug` which will post debug logs for every action that happens. So if you don't need it for a specific reason, you can drop the `--debug` flag.

**Reply by marefati110** [reply]
Yes, but I want to not send a message if there is not update to get 
even i drop --debug i gen t notifications

**Comment by simonaronsson** [comment, marked as answer]
That message is not just a "everything's ok, proceeding as usual", but rather a tell that something is wrong (the error return from the previous function call was not `nil`). For some reason, watchtower is unable to update your container. The most likely reason is that something has happened with the image the container was created from. We can't now for sure, however, as we currently don't print the actual error message.

This will be added in the next version of watchtower, but until then; would you mind recreating the containers you're monitoring to see whether the issue goes away?

Thanks,
Simme
